pkgs/development/tools/analysis/lcov/default.nix

```
@@ -16,9 +16,8 @@ let
     perlPackages.DevelCover
     perlPackages.GD
     perlPackages.JSONXS
-    perlPackages.MemoryProcess
     perlPackages.PathTools
-  ];
+  ] ++ lib.optional (!stdenv.isDarwin) [ perlPackages.MemoryProcess ];
```

```suggestion
  ] ++ lib.optionals (!stdenv.isDarwin) [ perlPackages.MemoryProcess ];
```

`lib.optional` adds an additional list resulting in `[ [ perlPackages.MemoryProcess ] ]` which is not what you want.

---

Thanks!